src/avram/charms/enum_extensions.cr

```diff
@@ -37,10 +37,6 @@ macro avram_enum(enum_name, &block)
       @enum = Avram{{ enum_name }}.from_value(enum_value.to_i)
     end
 
-    def ==(other : {{ enum_name }}) : Bool
-      self.enum == other.enum
-    end
-
     delegate :===, to_s, to_i, to: @enum
```

Comment:
Do we still need to delegate `===` here?

Reply:
Yep, the case equality tests fail without it.